pkg/git/git.go

```diff
@@ -104,11 +103,7 @@ func (git *Client) cloneRepoInPath(clonePath string, options CloneOptions) error
 		// Switch to target branch
 		args := []string{"checkout", options.Branch}
 		if options.Bootstrap {
-			empty, err := git.isRepoEmpty()
-			if err != nil {
-				return err
-			}
-			if empty {
+			if !git.isRemoteBranch(options.Branch) {
```

Comment:
It's not clear to me why it's important that it not be a remote branch in order to execute `checkout -b`? What happens if it isn't a remote branch but still exists? Why not check existence for the condition?

Reply:
:100: